packages/web/src/components/basic/ReactiveComponent.js

```diff
@@ -173,6 +175,7 @@ const mapStateToProps = (state, props) => ({
 		(state.selectedValues[props.componentId]
 			&& state.selectedValues[props.componentId].value)
 		|| null,
+	loading: state.isLoading[props.componentId],
```

Comment:
Should we rename it to `isLoading` everywhere?

Reply:
It won't effect anyways, since we're not exposing it to consumers.

Reply:
It improves the readabilty and meaning.